src/content/app/species/SpeciesPageContent.tsx

```diff
@@ -123,7 +123,9 @@ const TopBar = () => {
         <Chevron direction="left" animate={false} />
         <span className={styles.pageTitle}>Find a Species</span>
       </div>
-      <div className={styles.dataForSpecies}>Data for this species</div>
+      <div className={styles.dataForSpecies}>
+        {/* placeholder for future species data */}
+      </div>
```

Comment:
I'm curious if there is any value in keeping this here. Surely it will be very easy to recreate if we have to put some info here later on? Did Andrea say that she wants to put something in this space?

Reply:
Yes she said that there will be some text for this space. I notice we do this often in the code, in fact my comment is exactly the same as to what i saw in the code.